### DISCUSSION TITLE
Whisper API only transcribing first few seconds for audio recorded in safari browser in mac device.

### DISCUSSION THREAD

**Discussion by mangalprada** [openai/whisper#1938] [Q&A]
The audio is recorded in mp4 format as it's the only format supported in safari for audio recording.  

Let's there is an audio clip which is 10 secs long. I'm only getting transcription for the first 1 sec. And for the rest I get nothing. 

```
transcript = client.audio.transcriptions.create(
                model="whisper-1",
                file=audio_file
            )
```

**Comment by CopBox** [comment]
Please someone help us
